skt/kerneltree.py

```diff
@@ -117,14 +117,14 @@ def __git_cmd(self, *args, **kwargs):
                                        **kwargs)
         except subprocess.CalledProcessError as exc:
             logging.debug(exc.output)
-            raise(exc)
+            raise exc
 
-    def __git_cmd_pipe(self, input, *args, **kwargs):
+    def __git_cmd_pipe(self, iinput, *args, **kwargs):
         """
         Feed input to a git command and return its output.
 
         Args:
-            input:      Input to feed to the git command.
+            iinput:      Input to feed to the git command.
```

Comment:
While this fixes the pylint error, the new name of variable is unclear and prone to errors by oversight (it's easy to write additional code with `input` since having two `i` at the beginning is not something people expect). Can you use a different name that's still clear, maybe something like `command_input` or similar? Also, the indentation in docstring needs to be adjusted for the variable name to fit.